cpp/dolfin/fem/utils.cpp

```diff
@@ -643,21 +643,45 @@ fem::create_functionspace(ufc_function_space* (*fptr)(void),
   return V;
 }
 //-----------------------------------------------------------------------------
+void fem::check_coefficients(const fem::Form& form)
```

Comment:
This function name doesn't say anything much. It should be something like:
```
std::set<foo> unset_constants
```
or 
```
bool coefficients_all_set();
```
(and likely a member of `FormCoefficients`).